tests/vmi_tpm_test.go

```diff
@@ -83,3 +89,97 @@ var _ = Describe("[sig-compute]vTPM", func() {
 		})
 	})
 })
+
+var _ = Describe("[sig-storage]vTPM", func() {
+	var virtClient kubecli.KubevirtClient
+	var err error
+
+	BeforeEach(func() {
+		virtClient, err = kubecli.GetKubevirtClient()
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	Context("with persistent TPM VM option enabled", func() {
+		It("[Serial]should persist TPM secrets across migrations and restarts", func() {
```

Comment:
Would it be better to have two separate tests: one for `migrations` another one - for `restarts`? IMHO, debugging smaller tests is usually easier.

Reply:
That's a good point, I actually went back-and-forth on the idea.
I chose to do all in 1 test because I think there's value in testing stop-start after migrating.
However, it would also be nice to test migrating after start-stop... Maybe I can try to create a table with 2 "operations", where the second one can be nil, and then test 4 entries:
- start-stop + nil
- migration + nil
- start-stop + migration
- migration + start-stop
That would take a while to run but it's probably good to have anyway.

Reply:
Done, PTAL!

Reply:
Looks good. Is it intentional that you skipped `start-stop + nil` and `migration + nil`?

Reply:
Yeah. I can add the +nil cases but it feels redundant.